app/furniture/marketplace/marketplaces/edit.html.erb

```diff
@@ -1,12 +1,16 @@
 <%- breadcrumb :edit_marketplace, marketplace %>
 
-<div class="max-w-2xl self-stretch mx-auto">
+<div class="max-w-2xl self-stretch md:container md:mx-auto px-40">
   <%= render "form", marketplace: marketplace %>
-  <p class="flex justify-between my-5 flex-wrap">
+  <p class="flex my-5 flex-col">
```

Comment:
Mobile view got messed up while trying to fix the widescreen layout. Anyone got clues on how to fix it? Or Should I just revert these layout changes and fix the squished together links a different way?

<img width="612" alt="Screenshot 2023-04-02 at 6 10 51 PM" src="https://user-images.githubusercontent.com/16140873/229390531-49a3ba59-ec08-4ce8-98a5-71413f0c2512.png">

Reply:
To keep things simpler and moving along, I'd say revert the layout changes for now.

(I also have a plan to improve this entire Market configuration layout very soon, so I'll look into that then.)

Reply:
reverted 5e360b500247de8ec9fb80590604bf91deb8b9c2